CONTRIBUTING.md

```diff
@@ -118,3 +118,16 @@ not being invalidated correctly in some cases. To run a build with a new instanc
 ## The small print
 
 This project is an open source project, please act responsibly, be nice, polite and enjoy!
+
+## Frequently Asked Questions
+
+* IntelliJ fails to import Quarkus Maven project with `java.lang.OutOfMemoryError: GC overhead limit exceeded` 
+
+In IntelliJ IDEA (version older than `2019.2`) if you see problems in the Maven view claiming `java.lang.OutOfMemoryError: GC overhead limit exceeded` that means the project import failed.
+  
+To fix the issue, you need to update the Maven importing settings:  
+`Build, Execution, Deployment` > `Build Tools`> `Maven` > `Importing` > `VM options for importer`
+To import properly Quarkus you need to set the Max Heap Size to 1Go: `-Xmx1g`
```

Comment:
It really depends, sometimes it can be up to 3Gs

Reply:
Yeah, for me it's `768M` but I have heard of people talking about 2G or more. Perhaps you can make a note of it?